src/Pages/PassengerInfo/Accordian.js

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+function Accordian({ title }) {
+  const content = useRef(null);
+  const [height, setHeight] = useState(0);
+  const toggleAccordion = () => {
+    setHeight(height === 0 ? content.current.scrollHeight + 30 : 0);
+  };
+  const checkInfo = () => {};
+
+  return (
+    <AccordianAll>
+      <AccordionHeading>
+        <Container>
+          <TitleStyle>{title}</TitleStyle>
+          <button onClick={toggleAccordion}>Click</button>
+        </Container>
+      </AccordionHeading>
+
+      <AccrodionContent height={height} ref={content}>
+        <Container1>
+          <NameText>
+            <FristName>
+              승객 성(영문)<Required>필수 입력</Required>
+              <InputAll type="text" />
+            </FristName>
+            <SecondName>
+              승객 이름(영문)<Required>필수 입력</Required>
+              <InputAll type="text" />
+            </SecondName>
+          </NameText>
+
+          <AddInfo>
+            <Gender>
+              성별<Required>필수 입력</Required>
+              <BtnRadio>
+                <Ckbox name="성별" type="radio" value="남자" />
+                <GenderFont for="Ckbox">남자</GenderFont>
+                <Ckbox name="성별" type="radio" value="여자" />
+                <GenderFont for="Ckbox">여자</GenderFont>
+              </BtnRadio>
+            </Gender>
+            <Birth>
+              생년월일(YYYYMMDD 8자리)<Required>필수 입력</Required>
+              <InputAll type="text" />
+            </Birth>
+          </AddInfo>
+          <MemberNum>
+            회원번호
+            <Required>필수 입력</Required>
+            <InputAll type="text" />
+          </MemberNum>
+          <SummitBox>
+            <button type="button" onClick={checkInfo}>
+              확인
+            </button>
+          </SummitBox>
+        </Container1>
+      </AccrodionContent>
+    </AccordianAll>
+  );
+}
+const AccordianAll = styled.div`
```

Comment:
```suggestion
const Container = styled.div`
```
어차피 아코디언 컴포넌트 안쪽에 있는 내용들이라서 굳이 아코디언이라는 네이밍을 전부 붙여줄 필요는 없고 저는 가장 바깥쪽 감싸는 태그의 경우 container라고 고민 없이 이름 짓는 걸 선호하는 편입니다